### DISCUSSION TITLE
Update Checkbox value inside Openlayers Sidebar after we change a layer visibility using JS Code

### DISCUSSION THREAD

**Discussion by oabuthiab** [visgl/deck.gl#6996] [Q&A]
i use openlayers sidebar to manager my layers, however users can remove a layer using a button. 
once the button is clicked JS changes the visibility of this layer and removes it from the map , the code used is :

item.setVisible(false);  

this works however the checkbox inside sidebar remains unchanged (checked) even when the layer is no longer visibile, how can i refresh the vlaue of checkboxes (inside sidebar) knowing that openlayers generates different IDs for each box whenever its value is changed or a layer is added. 

is there a code (openlayers code) to update or refresh checkboxes inside sidebar. 

thank you

**Comment by Pessimistress** [comment]
Does this question have anything to do with deck.gl? OpenLayers repo is here: https://github.com/openlayers/openlayers

**Reply by oabuthiab** [reply]
I mistakenly added it here, thanks
